CHANGES.rst

```diff
@@ -4,7 +4,7 @@ Changes
 1.4.4 (unreleased)
 ------------------
 
-- Add support for PyPy3.
+- Add support for PyPy3 and Jython 2.7.
```

Comment:
Rather than document the not-yet-ready-for-prime-time Jython support, let's add a changelog entry about the `valuerefs` change here.